Client/trunk/ParaEngineClient/BlockEngine/BlockChunk.cpp

```diff
@@ -375,6 +375,8 @@ namespace ParaEngine
 		int32_t blockIndex = m_blockIndices[nIndex];
 		if(blockIndex >= 0)
 		{
+			RemoveBlockMaterial(CalcPackedBlockID(blockId_r), -1);
```

Comment:
needs to make this function extremely fast for chunks without external materials.  CalcPackedBlockID(blockId_r) is already there, avoid duplicate calc